data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report10_extra.py

```diff
@@ -727,24 +727,31 @@ def my_sql(table_name, column_name):
     query = JINJA_ENV.from_string("""
 
 with df_person AS (
-SELECT distinct person_id,
-state_of_residence_concept_id as state_id_1,
-state_of_residence_source_value
-
-FROM `{{project_id}}.{{rt_cdr_deid_clean}}.person_ext` p
-JOIN `{{project_id}}.{{reg_combine}}._mapping_src_hpos_to_allowed_states` ON value_source_concept_id=state_of_residence_concept_id
-JOIN `{{project_id}}.{{deid_sand}}.site_maskings` m ON hpo_id=src_hpo_id
-WHERE state_of_residence_concept_id IS NOT NULL
+    SELECT distinct person_id,
+    state_of_residence_concept_id as state_id_1,
+    state_of_residence_source_value
+
+    FROM `{{project_id}}.{{rt_cdr_deid_clean}}.person_ext` p
+    JOIN `{{project_id}}.{{reg_combine}}._mapping_src_hpos_to_allowed_states` ON value_source_concept_id=state_of_residence_concept_id
+    JOIN `{{project_id}}.{{deid_sand}}.site_maskings` m ON hpo_id=src_hpo_id
+    WHERE state_of_residence_concept_id IS NOT NULL
 ),
 
 df_omop AS (
-SELECT distinct person_id,ext.src_id,m.src_id,hpo_id,mhpo.src_hpo_id,m.hpo_id,
-State,mhpo.value_source_concept_id as state_id_2
-FROM `{{project_id}}.{{rt_cdr_deid_clean}}.{{table_name}}`
-JOIN `{{project_id}}.{{rt_cdr_deid_clean}}.{{table_name}}_ext` ext USING ({{column_name}})
-JOIN `{{project_id}}.{{deid_sand}}.site_maskings` m ON ext.src_id=m.src_id
-JOIN `{{project_id}}.{{reg_combine}}._mapping_src_hpos_to_allowed_states` mhpo ON mhpo.src_hpo_id=m.hpo_id
-WHERE ext.src_id !='PPI/PM'
```

Comment:
The meaning here is that  ext.src_id != 'PPI/PM' which has meant in the past REGEXP_CONTAINS(src_id, r'(?i)EHR site')

Suggestion: You might write the new statement as WHERE REGEXP_CONTAINS(src_id, r'(?i)EHR site')